src/ui_func.py

```diff
@@ -0,0 +1,106 @@
+from classes import SQLiteDataBase
+
+
+def show_menu():
+    global MENU
```

Comment:
you don't need to use global here because your are not changing MENU variable

Reply:
applied